packages/calcite-components/src/components/block/block.scss

```diff
@@ -66,20 +76,20 @@
 .toggle {
   @apply font-sans
     focus-base
-    m-0
     flex
     cursor-pointer
     flex-nowrap
     items-center
     justify-between
     border-none
+    m-0
     p-0;
 
   text-align: initial;
-  background-color: transparent;
+  background-color: var(--calcite-block-header-background-color, transparent);
```

Comment:
make sure `background-color: var(--calcite-block-header-background-color, transparent);` is being applied to the entire header as well.

Reply:
✅ checked!